packages/calcite-components/src/components/combobox-item/combobox-item.tsx

```diff
@@ -131,6 +131,12 @@ export class ComboboxItem extends LitElement implements InteractiveComponent {
    */
   @property() value: any;
 
+  /**
+   * Specifies whether the user set the hidden attribute in the HTML
+   *
+   */
+  @property() hideItem = false;
```

Comment:
Thanks for removing `user` from the prop name! To align with `hidden`, WDYT about `hiddenItem`?